src/Analyzer/ConfluenceAnalyzer.php

```diff
@@ -59,6 +68,31 @@ class ConfluenceAnalyzer extends AnalyzerBase implements LoggerAwareInterface, I
 	 */
 	private $pageConfluenceTitle = '';
 
+	/**
+	 * @var QuestionHelper
+	 */
+	private $questionHelper = null;
+
+	/**
+	 * @var string
+	 */
+	private $mainpage = 'Main Page';
+
+	/**
+	 * @var boolean
+	 */
+	private $extNsFileRepoCompat = false;
+
+	/**
+	 * @var array
+	 */
+	private $advancedConfig = [];
+
+	/**
+	 * @var bool
+	 */
+	private $hasAdvancedConfig = true;
```

Comment:
Shouldn't we assume that there is _no_ advanced config by default?

Reply:
Set to false by default